### PR TITLE
Update controller to create Service resources

### PR DESCRIPTION
**Pull request by tomocy**
This PR implements the feature to update a controller to reconcile services for primary and replica MySQL pods.

### CODE REVIEW

File: controllers/mysql_clustering_test.go [hunk 1]
```diff
@@ -80,13 +80,24 @@ func TestDecideNextOperation(t *testing.T) {
...
-						primaryHost: hostName(0),
+						Index:       2,
+						PrimaryHost: hostName(0),
 					},
+					&setLabelsOp{},
+				},
+			},
+		},
+		{
+			name:  "LabelsAreWrong",
```
Comment (kfyharukz) [inline controllers/mysql_clustering_test.go:95]: This test only covers the case "a read-only instance has a wrong label".
I think there is the case "a writable (it's the current primary) instance has a wrong label".  Isn't the test for this case needed?

**Review by kfyharukz — CHANGES_REQUESTED** [review]

**Comment by llamerada-jp** [comment]
Isn't the length check needed at line 36 at infrastructure.go.

**Comment by llamerada-jp** [comment]
Please remove `fmt.Println` at line 47 at mysql_accessor.go.

File: Makefile [hunk 2]
```diff
@@ -71,12 +74,15 @@ mod:
 
 setup:
 	$(SUDO) apt-get update
-	curl -sL https://go.kubebuilder.io/dl/$(KUBEBUILDER_VERSION)/$(GOOS)/$(GOARCH) | tar -xz -C /tmp/
-	$(SUDO) rm -rf /usr/local/kubebuilder
-	$(SUDO) mv /tmp/kubebuilder_$(KUBEBUILDER_VERSION)_$(GOOS)_$(GOARCH) /usr/local/kubebuilder
-	$(SUDO) curl -o /usr/local/kubebuilder/bin/kustomize -sL https://go.kubebuilder.io/kustomize/$(GOOS)/$(GOARCH)
-	$(SUDO) chmod a+x /usr/local/kubebuilder/bin/kustomize
+	mkdir -p bin
```
[the remark this answers is not in the document]
Comment (entooone) [reply Makefile:77]: `./bin` is intended to store binaries other than those of `kubebuilder` , so we don't use `KUBEBUILDER_ASSETS` .

[the remark this answers is not in the document]
Comment (llamerada-jp) [reply Makefile:77]: I see.

**Review by llamerada-jp — APPROVED** [review]
LGTM

**Review by kfyharukz — APPROVED** [review]
LGTM

**Pull request merged**